compiler/test/stdlib/rational.test.gr

```diff
@@ -0,0 +1,63 @@
+module RationalTest
+
+let (===) = (==)
+let (!==) = (!=)
```

Comment:
Looks like we need the same changes here as your other PR so you aren't shadowing `==` and `!=` in the entire file.